apps/opencs/model/world/columnbase.hpp

```diff
@@ -121,14 +131,32 @@ namespace CSMWorld
 
         virtual std::string getTitle() const;
 
-        virtual int getId() const;
+        virtual int getId() const; // FIXME: why have an accessor for a public member?
+    };
+
+    class NestableColumn : public ColumnBase
+    {
+        std::vector<NestableColumn *> mNestedColumns;
+        bool mHasChildren;
```

Comment:
You are correct. Unnecessary.  But I just like having members if they allow avoiding lots of function calls.